src/bastion/examples/hello_world.rs

```diff
@@ -9,39 +9,39 @@ fn main() {
     Bastion::init();
     Bastion::start();
     // We are creating a group of children
-    Bastion::children(|children| {
+    let workers = Bastion::children(|children| {
         // We are creating the function to exec
         children.with_exec(|ctx: BastionContext| {
             async move {
-                // We are creating the asker
-                let answer = ctx
-                    // It sending a message to the current child and return the Answer
-                    .ask(&ctx.current().addr(), "hello, world!")
-                    .expect("Couldn't send the message.");
                 // We are defining a behavior when a msg is received
                 msg! {
-                  // We are waiting a msg
-                  ctx.recv().await?,
-                  // We are catching a msg
-                  msg: &'static str =!> {
-                    // We are answering to a msg
-                    let _ = answer!(ctx, msg);
-                  };
-                  _: _ => ();
+                    // We are waiting a msg
+                    ctx.recv().await?,
+                    // We are catching a msg
+                    msg: &'static str =!> {
+                        // Printing the incoming msg
+                        println!("{}", msg);
+                        // Sending the asnwer
+                        answer!(ctx, msg).expect("couldn't reply :(");
+                    };
+                    _: _ => ();
                 }
-                // We are waiting the answer
-                let (msg, _) = answer.await?.extract();
-                // We are casting the received message into it waiting type
-                let msg: &str = msg.downcast().unwrap();
-                // We are printing the message
-                println!("{}", msg);
-                // We are stopping bastion here
-                Bastion::stop();
                 Ok(())
             }
         })
     })
     .expect("Couldn't create the children group.");
-    // We are waiting until the Bastion has stopped or got killed
-    Bastion::block_until_stopped();
+    // We are creating the asker
+    let asker = async {
+        // We are getting the first (and only) worker
+        let answer = workers.elems()[0]
+            .ask_anonymously("hello, world!")
+            .expect("Couldn't send the message.");
+        // We are waiting for the asnwer
+        answer.await.expect("couldn't receive answer");
+    };
+    // We are running the asker in the current blocked thread
+    run!(asker);
+    // We are stopping bastion here
+    Bastion::stop();
```

Comment:
Not sure if relevant, but I always      Bastion::block_until_stopped();  after Bastion::stop()

Reply:
` Bastion::block_until_stopped()` is to use before `Bastion::stop()`, because the first is waiting the second. So it's not really usefull to wait for stop after the stop.